glean-core/android/src/test/java/mozilla/telemetry/glean/private/EventMetricTypeTest.kt

```diff
@@ -450,12 +450,12 @@ class EventMetricTypeTest {
         // Trigger worker task to upload the pings in the background
         triggerWorkManager(context)
 
-        var request = server.takeRequest(20L, TimeUnit.SECONDS)
-        var docType = request.path.split("/")[3]
+        val request = server.takeRequest(20L, TimeUnit.SECONDS)
+        val docType = request.path.split("/")[3]
```

Comment:
what's the reason for this? no more mutability needed? or it was never needed, but it didn't produce warning/errors?

Reply:
It was never needed, for some reason it did not produce a warning at compile time, but it produced warnings within Android Studio